redbot/core/bot.py

```diff
@@ -1040,7 +1048,7 @@ async def load_extension(self, spec: ModuleSpec):
 
     def remove_cog(self, cogname: str):
         cog = self.get_cog(cogname)
-        if cog is None:
+        if cog is None or isinstance(cog, Core):
```

Comment:
Isn't this supposed to be checking for `_RuleDropper` (although in terms of cogs this is practically equivalent)?
```suggestion
        if cog is None or isinstance(cog, commands.commands._RuleDropper):
```